test/Crowdsale.js

```diff
@@ -1,5 +1,8 @@
+import moment from 'moment'
```

Comment:
Yes, that's what I was thinking too. Maybe we can roll our own small `duration` helper module. With e.g. `duration.weeks(1)` returning the appropriate number of seconds.

I'm just interested in removing the moment.js dependency from the project so as to avoid future developers using it in a way that could cause issues.